tests/test_rewrite.py

```diff
@@ -172,3 +172,28 @@ def test_curry_functor():
 
     rewriter = Rewriter([CurryRewriteRule()])
     assert rewriter(diagram).normal_form() == expected
+
+
+def test_unknown_words_rewrite_rule():
+    sentence = 'I see the unknown'
+    parser = BobcatParser()
+    diagram = parser.sentence2diagram(sentence).normal_form()
+
+    unknown_words = ['unknown']
+    rule = UnknownWordsRewriteRule(
+        template=diagram,
```

Comment:
The diagram that we are rewriting should not be being passed as a template, but this has no effect as `template` is never used in the `RewriteRule`.